test/regressions/tests/TextField/TextFieldMargin.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+import TextField from 'material-ui/TextField';
+
+export default function TextFieldMargin() {
```

Comment:
Also, that seems incorrect, there is no visual variations between the 3 inputs:
<img src="https://s3.amazonaws.com/argos-screenshots-us/d0e71e7879e47c80cabe7719e755ca81" />